.github/workflows/periodic.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       docker-image-name: pytorch-linux-bionic-cuda11.7-cudnn8-py3-gcc7
       test-matrix: |
         { include: [
-          { config: "default", shard: 1, num_shards: 2, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "default", shard: 1, num_shards: 2, runner: "linux.8xlarge.nvidia.gpu" },
```

Comment:
This might not help because 8xlarge runners just have 2 GPUs instead of 1.  The GPU spec is the same.  AFAIK, the test will only use the first GPU because this is not distributed tests.  Also this might cause unexpected behaviors when some tests detects the 2 GPUs and run in distributed mode.

IMO, a way to fix this would probably to run the periodic test sequentially